src/System.Net.Security/tests/UnitTests/System/Security/Authentication/ExtendedProtection/ExtendedProtectionPolicyTest.cs

```diff
@@ -53,33 +66,52 @@ public void ExtendedProtectionPolicy_OSSupportsExtendedProtection()
+            Assert.Equal(ProtectionScenario.TransportSelected, extendedProtectionPolicy.ProtectionScenario);
             Assert.Equal(customChannelBindingParam, extendedProtectionPolicy.CustomChannelBinding);
         }
 
         [Fact]
         public void ExtendedProtectionPolicy_ToString()
         {
-            var serviceName1 = "Test1";
-            var serviceName2 = "Test2";
-            var serviceNameCollectionParam = new ServiceNameCollection(new List<string> { serviceName1, serviceName2 });
-            var policyEnforcementParam = PolicyEnforcement.Always;
-            var protectionScenarioParam = ProtectionScenario.TransportSelected;
-            var extendedProtectionPolicy = new ExtendedProtectionPolicy(policyEnforcementParam, protectionScenarioParam, serviceNameCollectionParam);
-            var expectedString = $"ProtectionScenario={protectionScenarioParam}; PolicyEnforcement={policyEnforcementParam}; CustomChannelBinding=<null>; ServiceNames={serviceName1}, {serviceName2}";
-
-            var result = extendedProtectionPolicy.ToString();
-
-            Assert.NotNull(result);
-            Assert.Equal(expectedString, result);
+            string serviceName1 = "Test1";
+            string serviceName2 = "Test2";
+            ServiceNameCollection serviceNameCollectionParam = new ServiceNameCollection(new List<string> { serviceName1, serviceName2 });
+            ExtendedProtectionPolicy extendedProtectionPolicy = new ExtendedProtectionPolicy(PolicyEnforcement.Always, ProtectionScenario.TransportSelected, serviceNameCollectionParam);
+            string expectedResult = $"ProtectionScenario={ProtectionScenario.TransportSelected}; PolicyEnforcement={PolicyEnforcement.Always}; CustomChannelBinding=<null>; ServiceNames={serviceName1}, {serviceName2}";
+
+            string result = extendedProtectionPolicy.ToString();
+
+            Assert.Equal(expectedResult, result);
+        }
+
+        [Fact]
+        public void ExtendedProtectionPolicy_NoCustomServiceNamesAndNoCustomChannelBinding_ToString()
+        {
+            ExtendedProtectionPolicy extendedProtectionPolicy = new ExtendedProtectionPolicy(PolicyEnforcement.Always);
+            string expectedResult = $"ProtectionScenario={extendedProtectionPolicy.ProtectionScenario}; PolicyEnforcement={PolicyEnforcement.Always}; CustomChannelBinding=<null>; ServiceNames=<null>";
+
+            string result = extendedProtectionPolicy.ToString();
+
+            Assert.Equal(expectedResult, result);
+        }
+
+        [Fact]
+        public void ExtendedProtectionPolicy_NoCustomServiceNames_ToString()
+        {
+            MockCustomChannelBinding channelBinding = new MockCustomChannelBinding();
+            ExtendedProtectionPolicy extendedProtectionPolicy = new ExtendedProtectionPolicy(PolicyEnforcement.Always, channelBinding);
+            string expectedChannelBinding = channelBinding.ToString();
```

Comment:
you don't need to use a separate variable for 'expectedChannelBinding'.  Just use 'channelBinding.ToString()' in the below.